editor/test/reducer.js

```diff
@@ -921,6 +917,7 @@ describe( 'state', () => {
 
 	describe( 'userData()', () => {
 		beforeAll( () => {
+			require( '../library' );
```

Comment:
Same question as previous: We didn't need this previously? Or was it the case that because we imported from `@wordpress/blocks` we implicitly included the full library as well? 😬

Reply:
exactly

Reply:
The best thing to do is refactoring the test to not rely on these, but this PR is big enough :)

Reply:
I like it that we need to explicitly register those blocks. It just isn't clear from the require statement. We can revisit later.